x-pack/plugins/alerting/server/task_runner/create_execution_handler.test.ts

```diff
@@ -69,15 +73,55 @@ test('calls executeAction per selected action', async () => {
     `);
 });
 
-test('limits executeAction per action group', async () => {
+test(`doesn't call actionsPlugin.execute for disabled actionTypes`, async () => {
+  createExecutionHandlerParams.actionsPlugin.isActionTypeEnabled.mockReturnValueOnce(false);
+  createExecutionHandlerParams.actionsPlugin.isActionTypeEnabled.mockReturnValueOnce(true);
+  const executionHandler = createExecutionHandler({
+    ...createExecutionHandlerParams,
+    actions: [
+      ...createExecutionHandlerParams.actions,
+      {
+        id: '2',
+        group: 'default',
+        actionTypeId: 'test2',
+        params: {
+          foo: true,
+          contextVal: 'My other {{context.value}} goes here',
+          stateVal: 'My other {{state.value}} goes here',
+        },
+      },
+    ],
+  });
+  await executionHandler({
+    actionGroup: 'default',
+    state: {},
+    context: {},
+    alertInstanceId: '2',
+  });
+  expect(createExecutionHandlerParams.actionsPlugin.execute).toHaveBeenCalledTimes(1);
+  expect(createExecutionHandlerParams.actionsPlugin.execute).toHaveBeenCalledWith({
+    id: '2',
+    params: {
+      foo: true,
+      contextVal: 'My other  goes here',
+      stateVal: 'My other  goes here',
+    },
+    spaceId: 'default',
+    apiKey: createExecutionHandlerParams.apiKey,
+  });
+});
+
+test('limits actionsPlugin.execute per action group', async () => {
   const executionHandler = createExecutionHandler(createExecutionHandlerParams);
   await executionHandler({
     actionGroup: 'other-group',
     state: {},
     context: {},
     alertInstanceId: '2',
   });
-  expect(createExecutionHandlerParams.executeAction).toMatchInlineSnapshot(`[MockFunction]`);
+  expect(createExecutionHandlerParams.actionsPlugin.execute).toMatchInlineSnapshot(
```

Comment:
I ran into this test yesterday, and scratched my head.  I'm not sure it adds a ton of value.  I wonder if it did something different in the past, and then something changed, and someone updated the snapshot to match what the code was doing, or something ...

Reply:
Yeah, I use to do a lot of snapshots on functions to ensure # of calls + arguments match up. In this scenario it would of been to assert no calls went through. Though I moved towards using `.not.toHaveBeenCalled()` nowdays.. I will fix this while it's in context.

Reply:
Fixed in https://github.com/elastic/kibana/pull/59699/commits/a010126e755be848f10afa8843b236a54f9b2b38